natashasthilaire/Swampy-Sells
# User to post

### About
This PR is for the backend functionality of post a new listing, and also includes the alert messages being changed to use a react-library

### Test Instructions
1. Run `user_to_post` branch locally
2. Log in, navigate to post a listing, add listing, it should appear in the mongodb Items collection

### Related Tickets
[SS-39](https://swampysells.atlassian.net/browse/SS-39)
[SS-34](https://swampysells.atlassian.net/browse/SS-34)
